BrewServicesMenubar/AppDelegate.swift

```diff
@@ -26,9 +30,17 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     var services: [Service]?
 
     func applicationDidFinishLaunching(_ aNotification: Notification) {
-        UserDefaults.standard.register(defaults: [
-            brewExecutableKey: "/usr/local/bin/brew"
-        ])
+        // Homebrew can now be located in two different locations depending on how it was installed and what architecture the computer is running
+        // Define the most likely path first based on the architecture
+        #if arch(arm64)
```

Comment:
Is the architecture check necessary? I'm just wondering what the implications are if we have the same order for both platforms? e.g. `"/opt/Homebrew/bin/brew", "/usr/local/bin/brew"`?

Reply:
It's just a tiny optimization. We'd just do one unnecessary file existence check every time we try to invoke `brew`. The app should work fine without it.